iroha_client/config.json

```diff
@@ -0,0 +1,39 @@
+{
+  "TORII_URL": "127.0.0.1:1337",
+  "PUBLIC_KEY": {
+    "inner": [
+      207,
+      157,
+      30,
+      3,
```

Comment:
Does it have to be vertically formatted?

Reply:
Sry - I needed to switch over from VIM to VScode because I ruined my `vimrc` and it makes this auto formatting, I think we will need to replace it with hex as planned in multi hash keys issue